### PR TITLE
PLANNER-2435: Kogito task assigning phase1 blog post

### PR DESCRIPTION
**Pull request by wmedvede**

### CODE REVIEW

**Comment by wmedvede** [comment]
Hi @ge0ffrey ,  while some style and formatting adjustments are sill needed I think It's in a reasonably shape for an initial review.

Would you mind have look?

Thanks.

File: content/blog/2021/06/15/LetsOptaPlanYourJbpmTasksPart2.adoc [hunk 1]
```diff
@@ -0,0 +1,131 @@
+= Let's OptaPlan your jBPM tasks (part 2) - Moving to Kogito
+wmedvede
+2021-06-15
+:page-interpolate: true
+:jbake-type: post
+:jbake-tags: feature, task assignment, integration
+:jbake-social_media_share_image: CreditDisputeProcessDiagram.png
+
+In the previous blog post link:../../../2020/07/03/LetsOptaPlanYourJbpmTasksPart1.html[Let's OptaPlan your jBPM tasks (part 1)] we showed that OptaPlanner can be used for assigning the human tasks produced by a set of business processes to users according to an optimized plan.
```
Comment (ge0ffrey) [inline content/blog/2021/06/15/LetsOptaPlanYourJbpmTasksPart2.adoc:9]: Can we make the first paragraph more answer the question "What is the benefit for me reading this?" What is the value proposition? Why should I care?

Referencing the previous blog is good, but I 'd leave that to the second paragraph, if you want them to read that first to better understand the problem. Readers are lazy, the TLDR of the problem/value should be in the first paragraph.

Readers might not now what Kogito is or what the benefit of kogito is.

Maybe something in the lines of
"In BPMN workflows in the cloud, we can automatically assign tasks to users by combining Kogito processes with OptaPlanner optimization. Furthermore, we can optimize those assignments, to ensure every user has the right skills for the tasks and ..."

File: content/blog/2021/06/15/LetsOptaPlanYourJbpmTasksPart2.adoc [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+
+We can see that the task requires "a CreditAnalyst user with skills in the MASTER credit card". This is why it was assigned to katy.
+
+As soon this task is completed, a new instance of the Notify Customer task is created.
+
+The following image shows the new instance information:
+
+image::NotifyCustomer_1111_MASTER_Spanish_bob.png[]
+
+We can see that the Spanish language was selected as the "Preferred language for communications", which establishes that:
```
Comment (ge0ffrey) [inline content/blog/2021/06/15/LetsOptaPlanYourJbpmTasksPart2.adoc:119]:
```suggestion
Notice that the Spanish language was selected as the "Preferred language for communications", which establishes that:
```

File: content/blog/2021/06/15/LetsOptaPlanYourJbpmTasksPart2.adoc [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+|bob | ClientRelations | Spanish and English languages.
+|frank | ClientRelations | Chinese language.
+
+|===
+
+When the process is started a new instance of the ResolveDispute task is created and automatically assigned:
+
+image::ResovleDispute_1111_MASTER_Spanish_katy.png[]
+
+We can see that the task requires "a CreditAnalyst user with skills in the MASTER credit card". This is why it was assigned to katy.
```
Comment (ge0ffrey) [inline content/blog/2021/06/15/LetsOptaPlanYourJbpmTasksPart2.adoc:111]: katy with a capital?

Writing docs guidelines is use active tense, which works well in blogs too:
"we can see that the task requires" => "The task requires ..."

File: content/blog/2021/06/15/LetsOptaPlanYourJbpmTasksPart2.adoc [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+
+"The task requires a ClientRelations user with skills in the Spanish language", and this is why this new task is assigned to bob.
+
+As in the previous blog post the example above is quite linear, only four users are defined, and the possible assignments are direct. This was done intentionally to show how the assignment decisions are made accordingly with the configured users, groups, and the tasks business data.
+
+== Conclusions
+
+OptaPlanner can be used in different ways, ranging from stand-alone CLI to complex web applications like vehicle routing, Kie Server based applications, https://quarkus.io/guides/optaplanner[Quarkus] applications, and also as part of any https://kogito.kie.org/[Kogito] application.
+
+In general, to integrate OptaPlanner in your https://kogito.kie.org/[Kogito] application is straightforward, you instruct OptaPlanner with the model, and the Solver configuration to use, and that’s all ;), OptaPlanner link:../../../../compatibility/quarkus.html[Quarkus] or link:../../../../compatibility/springBoot.html[SpringBoot] extensions will manage all the complexity of the Solver instantiation and management.
```
Comment (ge0ffrey) [inline content/blog/2021/06/15/LetsOptaPlanYourJbpmTasksPart2.adoc:129]: "is straightforward, you instruct" => "is straightforward: you instruct"

**Review by ge0ffrey — APPROVED** [review]

**Comment by ge0ffrey** [comment]
Looks good. Thanks for the blog, Walter!

**Comment by wmedvede** [comment]
Hi @ge0ffrey , your suggestions where applied, the blog post is ready.

Important notes:

1. before merging, we need to update the blog post date/directory to the real publication date. (not a big deal)
2. This blog post refers to the Task Assigning Service documentation that is under @hmanwani-rh review here https://github.com/kiegroup/kie-docs/pull/3689 at this moment. So we should ideally merge that PR before.

Thanks!

**Comment by ge0ffrey** [comment]
@yurloc this is good to merge, can you work with Walter to sync the merging with the date?

**Comment by wmedvede** [comment]
For the record, blog post date was moved to the 26th July as discussed. I'll keep an eye to make this merged (or changed if required) before this date.

**Pull request merged**